scss/bitstyles/atoms/avatar/_settings.scss

```diff
@@ -1,6 +1,6 @@
 @use '../../tools/size';
 
 $sizes: (
-  'm': size.get('xl'),
-  'l': size.get('xl') + size.get('m'),
+  'm': size.get('m'),
```

Comment:
This makes a big difference to the size of these components! Let’s keep the changes to utility classes only please, no components (the components need to be updated only where they reference sizes that have had their names changed, like `xxxl` or `xxl`)